### PR TITLE
Add Docker and Postgres support

### PR DESCRIPTION
**Pull request by phuongdh**
Part of https://github.com/avalonmediasystem/avalon/issues/2757

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,16 @@
 `rails db:migrate`
 
 `rails hydra:server`
+
+# Quickstart development with Docker
+Docker provides an alternative way of setting up an Avalon Media System Development Environment in minutes without installing any dependencies beside Docker itself. It should be noted that the docker-compose.yml provided here is for development only and will be updated continually.
+* Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/)
+* ```git clone https://github.com/avalonmediasystem/tbd```
```
Comment (cjcolvar) [inline README.md]: Please change this to samvera-labs/avalon-bundle in anticipation of transfer to labs.

File: README.md [hunk 2]
```diff
@@ -9,3 +9,16 @@
...
+* ```cd tbd```
+* ```docker-compose up```
+* Try loading Avalon in your browser: ```localhost:3000```
+
+Avalon is served by Webrick in development mode so any changes will be picked up automatically. Running a Rails command inside the Avalon container is easy, for example, to run tests ```docker-compose exec avalon bash -c "RAILS_ENV=test bundle exec rspec"```.
```
Comment (cjcolvar) [inline README.md]: Should this say `puma` instead of `webrick`?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    volumes:
+      - solr:/opt/solr/server/solr/mycores
+    entrypoint:
+      - docker-entrypoint.sh
+      - solr-precreate
+      - avalon
+      - /opt/solr/avalon_conf
+    networks:
+      internal:
+  matterhorn:
```
Comment (cjcolvar) [inline docker-compose.yml]: Please remove this since we're going to start without matterhorn.

File: config/database.yml [hunk 4]
```diff
@@ -22,4 +22,5 @@ test:
 
 production:
   <<: *default
+  adapter: postgresql
   database: db/production.sqlite3
```
Comment (cjcolvar) [inline config/database.yml]: Is this the right database?  Also do we need to support the environment variable being passed in by docker here?

Comment (phuongdh) [reply config/database.yml]: Env is already supported.

File: docker-compose.yml (hunk 3 above)
Comment (phuongdh) [reply docker-compose.yml]: ✅

File: README.md (hunk 2 above)
Comment (phuongdh) [reply README.md]: ✅

File: README.md (hunk 1 above)
Comment (phuongdh) [reply README.md]: ✅

File: README.md [hunk 5]
```diff
@@ -9,3 +9,16 @@
 `rails db:migrate`
 
 `rails hydra:server`
+
+# Quickstart development with Docker
+Docker provides an alternative way of setting up an Avalon Media System Development Environment in minutes without installing any dependencies beside Docker itself. It should be noted that the docker-compose.yml provided here is for development only and will be updated continually.
+* Install [Docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/)
+* ```git clone https://github.com/samvera-labs/avalon-bundle```
+* ```cd tbd```
```
Comment (cjcolvar) [inline README.md]: Please change to `avalon-bundle`.

**Pull request merged**